### PR TITLE
162_find_peak_element

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 162_find_peak_element.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+
+class Solution:
+    def findPeakElement(self, nums: List[int]) -> int:
```
Comment (ryujumoto) [inline 162_find_peak_element.py:7]: Your program is too complicated. Modify it with reference to solution or discuss.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I revised the code. I make the way simple and add the are and time complexity

File: 162_find_peak_element.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# T(n) O(1)
```
Comment (ryujumoto) [inline 162_find_peak_element.py]: #time: O(logn)

File: 162_find_peak_element.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+from typing import List
+
+
+class Solution:
+    def findPeakElement(self, nums: List[int]) -> int:
+      
+        left = 0
+        right = len(nums) - 1
+        while left < right:
+            mid = (left + right) // 2
```
Comment (ryujumoto) [inline 162_find_peak_element.py]: mid = (left+right) // 2

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I wrote. Please check my code.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**